packages/Chem/src/package.ts

```diff
@@ -151,6 +151,27 @@ export async function getMorganFingerprints(molColumn: DG.Column): Promise<DG.Co
   }
 }
 
+//name: checkSmilesValidity
+//input: column col
+//output: double perc
+export async function checkSmilesValidity(col: DG.Column): Promise<number> {
```

Comment:
No reason to check the whole column, this will take too much time, we have agreed on just passing 10 random values